crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/migrations.rs

```diff
@@ -107,27 +121,58 @@ impl<'a> LedgerDBMigrator<'a> {
         }
         // Stop using the original ledger DB path, i.e drop locks
         drop(new_ledger_db);
+
+        // Now that the lock is gone drop the tables that were migrated
+        for table in tables_to_drop {
+            LedgerDB::drop_cf(
+                &RocksdbConfig::new(temp_db_path.path(), max_open_files),
+                Some(all_column_families.clone()),
+                &table,
+            )?;
+        }
```

Comment:
I don't believe that dropping tables is the responsibility of the migration manager. Each migration should decide what it wants to do whether it's to migrate old tables to new or remove tables .. etc. Please call `drop_column_family` inside the migration itself.

Reply:
I don't understand that.

- The migration `execute` receives an instance of the `new_ledger_db` which is already using the one in tempdir.
- Do step number 3
- Drop cf

Reply:
I cannot drop cf without dropping all instances of a rocksdb

Reply:
What happens if you don't drop it?

Reply:
1) The ledger db will not open with the usual LEDGER_TABLES and cry missing column family etc etc because the struct is removed from LEDGER_TABLES
2) It stays there and occupies extra space

Reply:
Okay. Let me take a stab at it tomorrow morning and i'll let you know the result.

Reply:
> I cannot drop cf without dropping all instances of a rocksdb

I see your point now. The problem is that `DB` used by ledger is behind an `Arc` and `drop_cf` needs to borrow a mutable reference. Which is why we need to call `drop_cf` with the instance of `mut DB`